addons/mail/static/src/utils/common/format.js

```diff
@@ -6,7 +6,7 @@ import { escape, unaccent } from "@web/core/utils/strings";
 import { url } from "@web/core/utils/urls";
 
 const urlRegexp =
-    /\b(?:https?:\/\/\d{1,3}(?:\.\d{1,3}){3}|(?:https?:\/\/|(?:www\.))[-a-z0-9@:%._+~#=\u00C0-\u024F\u1E00-\u1EFF]{2,256}\.[a-z]{2,13})\b(?:[-a-z0-9@:%_+~#?&[\]^|{}`\\'$//=\u00C0-\u024F\u1E00-\u1EFF]|,(?!$| )|\.(?!$| |\.)|;(?!$| ))*/gi;
+    /\b(?:https?:\/\/\d{1,3}(?:\.\d{1,3}){3}|(?:https?:\/\/|(?:www\.))[-a-z0-9@:%._+~#=\u00C0-\u024F\u1E00-\u1EFF]{2,256}\.[a-z]{2,13})\b(?:[-a-z0-9@:%_+~#?&[\]^|{}`\\'$//=\u00C0-\u024F\u1E00-\u1EFF]|[.]*[a-z]|,(?!$| )|\.(?!$| |\.)|;(?!$| ))*/gi;
```

Comment:
This is a bit too naive, what is dots are followed by digits, or any other supported characters?
If github syntax was for example
`https://github.com/odoo/enterprise/compare/16.0...16.0-voip-fix_demo_data-tsm`
we would still want it to work (this is a possible example if we made our branches on the same repo).
So `[-a-z0-9@:%._+~#=\u00C0-\u024F\u1E00-\u1EFF]` or `[-a-z0-9@:%_+~#?&[\]^|{}`\\'$//=\u00C0-\u024F\u1E00-\u1EFF]`?
I'm not sure why we have 2 different parents

Reply:
> So [-a-z0-9@:%._+~#=\u00C0-\u024F\u1E00-\u1EFF] or [-a-z0-9@:%_+~#?&[\]^|{}`\\'$//=\u00C0-\u024F\u1E00-\u1EFF]?
>I'm not sure why we have 2 different parts like that.

The first group covers the domain name, this one is required, the other one is for the rest of the Url and is optional.